code/modules/mob/living/simple_animal/friendly/snake.dm

```diff
@@ -34,6 +34,7 @@
 	mob_size = MOB_SIZE_SMALL
 	mob_biotypes = MOB_ORGANIC|MOB_BEAST|MOB_REPTILE
 	gold_core_spawnable = HOSTILE_SPAWN
+	var/obj/item/udder/snake/udder
```

Comment:
I would suggest changing this to something like the snake's fang, as that's actually what's being interacted with. Not...udders. Bro...